From 6895ba1228a99bddf1f9ccf3a33cae35b7a1877d Mon Sep 17 00:00:00 2001
From: piyushk <piyushk@gmail.com>
Date: Thu, 2 May 2013 15:04:09 -0500
Subject: [PATCH 3/5] use share/libfreenect instead of share/PROJECT (#2)

---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0f47879..86d10ba 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -204,4 +204,4 @@ ENDIF ( BUILD_CPACK )
 ##############################
 
 # Install catkin package.xml
-install(FILES package.xml DESTINATION share/${PROJECT})
+install(FILES package.xml DESTINATION share/libfreenect)
-- 
1.7.9.5

